util.go

```diff
@@ -51,3 +55,52 @@ func BashComplete(c *cli.Context) {
 		PrintFlagNames(c.App.VisibleFlags(), c.App.Writer)
 	}
 }
+
+func ConfigPath() (string, error) {
+	// default config file path in `/etc/rhc/config.toml`
+	filePath := filepath.Join("/etc", LongName, "config.toml")
+
+	if _, err := os.Stat(filePath); os.IsNotExist(err) {
+		return "", nil
+	}
+
+	return filePath, nil
+}
+
+func GuessAPIURL() (string, error) {
+	var uString string
+	var baseURL *url.URL
+	// First check if the server api is set in conf
+	if DefaultConfig.APIServer != "" {
+		base, err := url.Parse("https://" + DefaultConfig.APIServer)
```

Comment:
I have tried following values of `base_url` in `/etc/insights-client/insights-client.conf`:

```
# Just un-commented default value that I found on RHEL9
base_url=cert-api.access.redhat.com:443/r/insights
```

```
# Valid URL
base_url=https://cert.console.redhat.com
```

But only following worked for me: `base_url=cert.console.redhat.com`.

Reply:
The default value in `insights-client` is not update, so is expected to not work